coalib/settings/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Has Setting, Section class; indexing tools; input checks; loads bears, files.
+
+- **class Setting:** Aggregates the settings passed to coala.
```

Comment:
No listing members. That is redundant to the TOC autogenerated from members, and it gets out of date quickly

Reply:
So remove the listing?

Reply:
yes, replace with prose